index.ts

```diff
@@ -132,7 +136,10 @@ function getTSConfig(options: Options, fileName: string): Options {
 		options.rootDir = configParseResult.options.rootDir;
 	}
 	options.files = configParseResult.fileNames;
-	return options;
```

Comment:
I killed it because it's not used, so it seemed clearer to return nothing.  Definitely the design here needs an overhaul.